### DISCUSSION TITLE
frquency for security scan

### DISCUSSION THREAD

**Discussion by tyuvara** [community/community#38133] [Code Security]
Hi, what frequency does security scan run's in github?

**Comment by ossanna16** [comment]
Hi there @tyuvara and welcome to our community! Thank you for asking a great question 🙂
While you are waiting for a reply to your question, here are a few things you can do:

To get started, introduce yourself in our [official introduction thread](https://github.com/orgs/community/discussions/23204)
To connect with other new users, check out this Discussion to [Meet Your GitHub Community Team](https://github.com/orgs/community/discussions/28776), further your GitHub knowledge by working through our [GitHub Skill 🆙 exercises](https://github.com/orgs/community/discussions/32121), and participate in our [weekly Fun Friday chats](https://github.com/orgs/community/discussions/30912)
